src/tribler/core/utilities/tracker_utils.py

```diff
@@ -132,6 +140,12 @@ def parse_tracker_url(tracker_url):
     if url.scheme == 'http' and not url.path:
         raise MalformedTrackerURLException(f'Missing announce path for HTTP tracker url ({tracker_url}).')
 
+    if url.scheme == 'https' and not url.port:
+        return url.scheme, (url.hostname, 443), url.path
```

Comment:
The number 443 is hardcoded here. Consider changing it to:
```suggestion
        return url.scheme, (url.hostname, HTTPS_PORT), url.path
```

Reply:
HTTPS_PORT number is extracted